ci/pipelines/Build.jenkinsfile

```diff
@@ -103,7 +91,7 @@ pipeline
 	{
 		always
 		{
-			junit allowEmptyResults: true, testResults: "build*/HashUtilsTest.xml"
+			junit allowEmptyResults: true, testResults: 'build*/test_reports/*.xml'
```

Comment:
Wouldn't be easier to save the test output on its repsective build folders `${buildFolder}/bin/${config}` and gather 'build*/**/*.xml ?

Reply:
It is a matter of choice. The idea behind test_reports folder is to not mix build products with execution products, but works both ways.

Reply:
Makes sense, that's good